fuse_tutorials/src/range_sensor_simulator.cpp

```diff
@@ -190,7 +193,8 @@ nav_msgs::Odometry::ConstPtr robotToOdometry(const Robot& state)
  *
  * The state estimator will not run until it has been sent a starting pose.
  */
-void initializeStateEstimation(const Robot& state)
+void initializeStateEstimation(
+  const Robot& state, rclcpp::Clock::SharedPtr clock=nullptr)
```

Comment:
Since this is an internal function, same comment as `beaconsToPointcloud` about unconditionally taking a clock. I see that where it's used [it's called without a clock](https://github.com/locusrobotics/fuse/blob/c18049be8cda5cf7ee3dd4eac7f3e2f1da50578e/fuse_tutorials/src/range_sensor_simulator.cpp#L380), but it could be given `*node->get_clock()` there.